frontend/src/components/city_table/city_table.tsx

```diff
@@ -41,13 +48,18 @@ export const CityTable: React.FC = () => {
     ) {
       setCityRows((cityRows) => [...cityRows, apiData]);
       setIsFull(false);
-    } else setIsFull(true);
+    } else if (cityRows.length === 5) {
+      setIsFull(true);
+      setSearchString("Limit 5 Cities Reached");
```

Comment:
Could be better to disable the input rather than display this string, shouldn't be too difficult to set a variable to the disabled attribute, thoughts?

Reply:
...just scrolled down, ignore me!